ui/cryptomaterial/dropdown.go

```diff
@@ -90,6 +92,10 @@ func (d *DropDown) Selected() string {
 	return d.items[d.SelectedIndex()].Text
 }
 
+func (d *DropDown) IsIndexChanged() bool {
+	return d.selectedIndex != d.preSelectedIndex
+}
```

Comment:
If we really need to know if the dropdown changed, we should have a `d.change bool`. We set this value to true when another item is selected.  When this method is executed, we read the old value(we'll return that to the caller if true) and then immediately set it to false (this is a common pattern in other places where we need to know if the dropdown changed).

Reply:
@ukane-philemon 

> When this method is executed
you mean Changed method? and changed status will return with Changed method? after that we set change = false again?

Reply:
Yes. Be sure to copy the initial value before resetting.

Reply:
Some cases we need to know old value and check changed outside Changed function, so I think current my code can resolve that case

Reply:
> so I think current my code can resolve that case

What you have currently will continue to return true if the dropdown was changed only once.

Reply:
> > so I think current my code can resolve that case
> 
> What you have currently will continue to return true if the dropdown was changed only once.

I don't think so, I have tested it worked

Reply:
Yh, works as expected.